### ISSUE TITLE
Fill the hands-on tutorial and the slides for main ChIP-seq tutorial and add a Galaxy tour

### ISSUE THREAD

**Issue by bebatut** [galaxyproject/training-material#92]
Hi, 

I started to fill the [tutorial](https://github.com/bgruening/training-material/blob/master/ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.md) and the [associated slides](https://github.com/bgruening/training-material/blob/master/ChIP-Seq/tutorials/estrogen_receptor_binding_site_identification.html) for ChIP-seq with [previous tutorial content](https://github.com/bgruening/training-material/blob/master/ChIP-Seq/tmp/ChIPseq.md).

We need to end to fill it with [previous tutorial content](https://github.com/bgruening/training-material/blob/master/ChIP-Seq/tmp/ChIPseq.md) by formatting the content to new templates.
We can also take inspiration from:
- https://drive.google.com/file/d/0B9urRnOAUUI8UmwzbTVpdmZucWM/view
- https://drive.google.com/file/d/0B9urRnOAUUI8cHpzYVBscjNKWEE/view

We also need to add a Galaxy tour

Thanks,

Bérénice

**Comment by malloryfreeberg** [comment]
Hi @bebatut!

I and my colleague @moheydarian are interested in contributing to tutorial content, including a ChIP-seq analysis tutorial. We have a curated set of published ChIP-seq data, workflows, etc. that have worked well for past training sessions. Would it be alright to format our content to the new templates? Or would it be preferred to keep the current ChIP-seq training data?

Either way, we are excited to contribute :)

**Comment by friedue** [comment]
Hi Mallory,
just curious: what data do you use? Does it cover a range of QC/peak types etc.?
The initial ChIP-seq training set is pretty old and shallowly sequenced, so updated material may not be a bad idea.

**Comment by bgruening** [comment]
@malloryfreeberg both should work. We are currently not sure how to handle multiple tutorials. I hope we can merge as many things we can and have one really good tutorial. But if we have two separated ones this is also good - in this case it is key to communicate to our users which tutorial does what, what are the difference and when do we recommend the one or the other. Does this makes sense?

We will also move the slides to pure markdown soon, so don't spend time in creating the reveal.js slides, we will auto generate those. More about this will follow soon.

Great to have you on board @malloryfreeberg! Greetings to Mo!

**Comment by malloryfreeberg** [comment]
Hi @friedue,

We are using Tal1 ChIP-seq data (with replicates and input controls) from [GSE51338](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51338) in GEO. There is also corresponding RNA-seq data in this study that we would like to incorporate somehow. We have downsampled the ChIP-seq data to include a subset of interesting loci identified in the associated [paper](https://www.ncbi.nlm.nih.gov/pubmed/25319994).

@bebatut, we can certainly work on creating tutorial material with both the previous ChIP-seq data and this updated data, and then later decide if having both tutorial is worthwhile.

**Comment by friedue** [comment]
@malloryfreeberg do you have your current tutorial online somewhere? sounds like a good data set (i.e., often used model organism, solid data, good results)

**Comment by malloryfreeberg** [comment]
@friedue not yet. @moheydarian and I were going to create a Galaxy Page tutorial today, and then came across all the training materials here. So, rather than just create a Page (which we still want to do), we thought we could directly contribute to this repo.

**Comment by friedue** [comment]
do you use deepTools, too? I see you're in Baltimore - I'm up in New York, we could also have a telephone chat if that's quicker than an online discussion (and if that's ok with @bebatut - I've just taken the liberty here because most of the links point to material I've set up or contributed to)
btw, the link to the previous tutorial is broken @bebatut

**Comment by MoHeydarian** [comment]
Hi @friedue,
We haven't incorporated deepTools in our version of the chIPseq exercise, yet and are open to doing so. Is there a feature of deepTools you would like to see incorporated? 

A telephone chat is fine, but I would vote for keeping this discussion thread online so others can follow and contribute.

**Comment by bgruening** [comment]
@MoHeydarian all? ;)
Seriously, the QC and heatmap plotting is unmatched imho and is every time a `wow effect` during workshops.

**Comment by bebatut** [comment]
Hi all,

@malloryfreeberg Welcome aboard. Thanks!

As @bgruening mentioned, things will change (a bit) soon (before the end of the week, I hope). Everything will be based on markdown (tutorials and slides). I will keep in touch to explain you the changes :)

@friedue I will fix the broken links during the changes. 

Thanks again

**Comment by friedue** [comment]
absolutely, I would leave this thread open.

deepTools has numerous features that are particularly helpful for ChIP-seq, most importantly QC (correlation, coverage, enrichment) and normalization (for sequencing depth, vs. input)
as Bjoern mentioned they also have a useful visualization features.
any time you'd prefer for a phone call today, @malloryfreeberg @MoHeydarian?

so many kudos for basing everything on markdown, @bebatut and @bgruening!!

**Comment by MoHeydarian** [comment]
Anytime this afternoon works for us, @friedue. Lets do a public Google hangout so others can join if they like (link: https://hangouts.google.com/call/kjpev2h7b5h5ho2a7xjrhu3ro4e) and lets aim to chat sooner than later in case our friends in other timezones would like to join.

**Comment by friedue** [comment]
2pm EST?

**Comment by MoHeydarian** [comment]
Sounds good for @malloryfreeberg and I. Talk soon.

**Comment by bgruening** [comment]
I need to leave soon, sorry will not be able to join! Enjoy and rock the training world!

**Comment by friedue** [comment]
stuck in a presentation, will be 10 min late

**Comment by friedue** [comment]
nice chat yesterday!
to get a feeling for how we typically use deepTools for ChIP-seq, you can check out [this page](http://deeptools.readthedocs.io/en/latest/content/example_usage.html#how-we-use-deeptools-for-chip-seq-analyses)

we also have a couple of [step-by-step protocols](http://deeptools.readthedocs.io/en/latest/content/example_step_by_step.html)

**Comment by malloryfreeberg** [comment]
Thanks for the chat, @friedue! We've implemented some deepTools functions in our workflow, but we are observing a strange trend in the plotFingerprint graph. Perhaps we can share our history/workflow with you today and get some feedback (on plotFingerprint or anything else)?

**Comment by friedue** [comment]
yes, totally. frd2007 at med dot cornell dot edu

**Comment by malloryfreeberg** [comment]
@friedue is that your Galaxy Main username? I can't find you to share the workflow :(

**Comment by friedue** [comment]
galaxy user name is friedue, but do send me an email for the back and forth about the workflow that's probably going to follow (slack's fine, too; same email address)

**Comment by malloryfreeberg** [comment]
Hey @bgruening, can you make a new branch under "training-material" called "chip-seq-tutorial-mfreeberg" or something similar, so that I can create a PR with which to merge mine and Mo's updated ChIP-seq tutorial branch? At least, I think this is how this is supposed to work... Thanks!

**Comment by bgruening** [comment]
@malloryfreeberg how about I add you all to this repo? :)

**Comment by bgruening** [comment]
Done! :)
Thanks so much for contributing!

**Comment by bebatut** [comment]
We now have 2 tutorials for ChIP-seq we can work on: http://galaxyproject.github.io/training-material/topics/chip-seq/
